src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -411,6 +411,8 @@ export default {
 					client.query({ query: experimentQuery, variables: { id: 'hover_loan_cards' } }),
 					// experiment: CASH-1113 Hover Loan Card Experiment
 					client.query({ query: experimentQuery, variables: { id: 'featured_sector' } }),
+					// experiment: CASH-1807 Recommendation Row Category Experiment (forthcoming)
+					client.query({ query: experimentQuery, variables: { id: 'recommendation_channel' } }),
```

Comment:
This depends upon you having included the uiexp.recommendation_channel in the lendByCategory.graphql as mentioned below.

Reply:
added recommendationCategoryRow: uiExperimentSetting(key: "recommendation_channel") to lendByCategory.graphql